pages/aboutUs.html

```diff
@@ -3,7 +3,7 @@
 <head>
   <meta charset='utf-8'>
   <meta http-equiv='X-UA-Compatible' content='IE=edge'>
-  <title>title</title>
+  <title>BiHjoGrafen</title>
```

Comment:
Maybe make it more specific? Like "BiHjoGrafen - Om oss"

Reply:
Good point, I will fix that!